<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/signal.png</normaloff>icons/signal.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_signal"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSave_signal_as"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuSelect_signal">
     <property name="title">
      <string>Select signal</string>
     </property>
     <addaction name="actionSignal1_2"/>
     <addaction name="actionSignal2_2"/>
     <addaction name="actionSignal3"/>
    </widget>
    <addaction name="ClearSignal"/>
    <addaction name="AddChannel"/>
    <addaction name="DeleteChannel"/>
    <addaction name="menuSelect_signal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="OpenSignalBtn"/>
   <addaction name="ZoomIn"/>
   <addaction name="ZoomOut"/>
   <addaction name="Beginning"/>
   <addaction name="PlayBtn"/>
   <addaction name="End"/>
   <addaction name="Pause"/>
   <addaction name="AddPanel"/>
   <addaction name="DeletePanel"/>
   <addaction name="spectrogram"/>
   <addaction name="PDF"/>
   <addaction name="LeftScroll"/>
   <addaction name="RightScroll"/>
  </widget>
  <action name="actionOpen_signal">
   <property name="text">
    <string>Open signal</string>
   </property>
  </action>
  <action name="actionSave_signal_as">
   <property name="text">
    <string>Save signal as </string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="OpenSignalBtn">
   <property name="icon">
    <iconset>
     <normaloff>icons/open-folder-with-document.png</normaloff>icons/open-folder-with-document.png</iconset>
   </property>
   <property name="text">
    <string>Open Signal</string>
   </property>
   <property name="toolTip">
    <string>Open Signal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="ZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>icons/zoom-in.png</normaloff>icons/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="toolTip">
    <string>ZoomIn</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="ZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>icons/zoom-out.png</normaloff>icons/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="toolTip">
    <string>ZoomOut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="Beginning">
   <property name="icon">
    <iconset>
     <normaloff>icons/backward.png</normaloff>icons/backward.png</iconset>
   </property>
   <property name="text">
    <string>Beggining</string>
   </property>
   <property name="toolTip">
    <string>Beggining</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </action>
  <action name="PlayBtn">
   <property name="icon">
    <iconset>
     <normaloff>icons/play-button.png</normaloff>icons/play-button.png</iconset>
   </property>
   <property name="text">
    <string>Play signal</string>
   </property>
   <property name="toolTip">
    <string>Play signal</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="End">
   <property name="icon">
    <iconset>
     <normaloff>icons/forward.png</normaloff>icons/forward.png</iconset>
   </property>
   <property name="text">
    <string>End</string>
   </property>
   <property name="toolTip">
    <string>End</string>
   </property>
   <property name="shortcut">
    <string>End</string>
   </property>
  </action>
  <action name="Pause">
   <property name="icon">
    <iconset>
     <normaloff>icons/pause.png</normaloff>icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="spectrogram">
   <property name="icon">
    <iconset>
     <normaloff>icons/Spectrogram.png</normaloff>icons/Spectrogram.png</iconset>
   </property>
   <property name="text">
    <string>Spectrogram</string>
   </property>
   <property name="toolTip">
    <string>Spectrogram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="PDF">
   <property name="icon">
    <iconset>
     <normaloff>icons/pdf.png</normaloff>icons/pdf.png</iconset>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
   <property name="toolTip">
    <string>PDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="ClearSignal">
   <property name="text">
    <string>Clear Signal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="AddChannel">
   <property name="text">
    <string>Add Channel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="DeleteChannel">
   <property name="text">
    <string>Delete Channel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="AddPanel">
   <property name="icon">
    <iconset>
     <normaloff>icons/addChannel.png</normaloff>icons/addChannel.png</iconset>
   </property>
   <property name="text">
    <string>Add panel</string>
   </property>
   <property name="toolTip">
    <string>Add Panel</string>
   </property>
  </action>
  <action name="DeletePanel">
   <property name="icon">
    <iconset>
     <normaloff>icons/delete.png</normaloff>icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Panel</string>
   </property>
   <property name="toolTip">
    <string>Delete Panel</string>
   </property>
  </action>
  <action name="LeftScroll">
   <property name="icon">
    <iconset>
     <normaloff>icons/left.png</normaloff>icons/left.png</iconset>
   </property>
   <property name="text">
    <string>Scroll Left</string>
   </property>
   <property name="toolTip">
    <string>Scroll Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="RightScroll">
   <property name="icon">
    <iconset>
     <normaloff>icons/right.png</normaloff>icons/right.png</iconset>
   </property>
   <property name="text">
    <string>Right Scroll</string>
   </property>
   <property name="toolTip">
    <string>Right Scroll</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionSignal2">
   <property name="text">
    <string>Signal2</string>
   </property>
  </action>
  <action name="actionSignal1_2">
   <property name="text">
    <string>Signal1</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionSignal2_2">
   <property name="text">
    <string>Signal2</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionSignal3">
   <property name="text">
    <string>Signal3</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
